questions/qa-http-and-lang.de.html

```diff
@@ -76,7 +76,7 @@ <h2><code class="kw" translate="no">meta</code>-Element</h2>
 
 <section id="http_summary">
 <h2>HTTP-Header</h2>
-  <p>Wenn Sie eine Website oder -ressource von einem Server anfordern, sendet der Server verschiedene Informationen zu dem Objekt zurück, das Sie anfordern (Metadaten). Er benutzt dabei ein Format, das als <a class="print" href="http://tools.ietf.org/html/rfc2616#section-14.12">HTTP-Header</a> bezeichnet wird. Eines der Dinge, die Sie in solchen Metadaten finden können, ist sprachbezogen. Achten Sie auf die letzte Zeile im folgenden Beispiel, welches die zu diesem Artikel gehörende HTTP-Antwort zeigt.</p>
+  <p>Wenn Sie eine Website oder -ressource von einem Server anfordern, sendet der Server verschiedene Informationen zu dem Objekt zurück, das Sie anfordern (Metadaten). Er benutzt dabei ein Format, das als <a class="print" href="https://httpwg.org/specs/rfc9110.html#field.content-language">HTTP-Header</a> bezeichnet wird. Eines der Dinge, die Sie in solchen Metadaten finden können, ist sprachbezogen. Achten Sie auf die letzte Zeile im folgenden Beispiel, welches die zu diesem Artikel gehörende HTTP-Antwort zeigt.</p>
```

Comment:
Why `httpwg.org` instead of the more official `rfc-editor.org`? I believe the latter is the official site (per John Kleinsin)?

Reply:
In my impression, the usability of `httpwg.org` is better. See also https://github.com/w3c/browser-specs/issues/280 and https://github.com/tobie/specref/issues/672